File: branches/Andean/Fantasy.Base/ServiceModel/ChannelSettings.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using Fantasy.Configuration;

namespace Fantasy.ServiceModel
{
    public class ChannelSettings : SettingsBase
    {
        private static ChannelSettings _default = (ChannelSettings)Fantasy.Configuration.SettingStorage.Load(new ChannelSettings());

        public static ChannelSettings Default
        {
            get
            {
                return _default;
            }
        }

        
        public List<AddressSetting> Addresses
        {
            get
            {
                return (List<AddressSetting>)base.GetValue("Addresses") ?? new List<AddressSetting>();
            }
        }
    }
}

File: branches/Stheno/AutoConverted/Fantasy.Base/ServiceModel/ChannelSettings.java
package Fantasy.ServiceModel;

import Fantasy.Configuration.*;

public class ChannelSettings extends SettingsBase
{
	private static ChannelSettings _default = (ChannelSettings)Fantasy.Configuration.SettingStorage.Load(new ChannelSettings());

	public static ChannelSettings getDefault()
	{
		return _default;
	}


	public final java.util.ArrayList<AddressSetting> getAddresses()
	{
		return (((java.util.ArrayList<AddressSetting>)super.GetValue("Addresses")) != null) ? (java.util.ArrayList<AddressSetting>)super.GetValue("Addresses") : new java.util.ArrayList<AddressSetting>();
	}
}